routers/router.mad1.yml

```
@@ -9,3 +9,13 @@
     remote_address: vpn.diogopassos.pt
     remote_port: 22097
     public_key: czJoGDj8855EvO0nwqVLH1Bv/I6quSaD1mddFE408nE=
+- name: ARFNET
```

```suggestion

- name: ARFNET
```

Small nit: add an empty line above to separate the peers.